serverless.yml

```diff
@@ -87,7 +87,6 @@ functions:
       SENTRY_ENVIRONMENT: ${self:provider.stage}
       SENTRY_RELEASE: ${env:CIRCLE_SHA1}
       FOLLOW_ON_IS_EMERGENCY_FIELD_ENABLED: ${self:custom.follow-on-is-emergency-field-enabled.${self:provider.stage}}
-      REPAIRS_FINDER_INTEGRATION_ENABLED: ${ssm:/repairs-hub/${self:provider.stage}/enable-repairs-finder-integration}
```

Comment:
What was this for?

Reply:
I had to replace it with a hardcoded ENV. Where its used, the user would see a visible change as the FT request is loaded